src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -349,159 +336,117 @@ public void testVisitLeaveToken()
+    public void testNonTightHtmlTagIntolerantCheckVisitCount() throws Exception {
         final String[] expected = {
-            "12: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
-            "19: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
-            "22: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "li"),
-            "28: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
-            "35: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "tr"),
-            "43: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "li"),
-            "51:8: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
-            "65: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "li"),
-            "74:13: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
-            "84: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
-            "94: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "tr"),
-            "114:8: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
-            "121:8: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
-            "125:8: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
-            "142: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
+            "13: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
+            "20: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
+            "23: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "li"),
+            "29: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
+            "36: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "tr"),
+            "44: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "li"),
+            "52:8: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "66: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "li"),
+            "75:13: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "85: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
+            "95: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "tr"),
+            "115:8: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "122:8: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "126:8: " + getCheckMessage(MSG_SUMMARY_FIRST_SENTENCE),
+            "143: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
         };
-        verify(checkConfig, getPath("InputAbstractJavadocNonTightHtmlTagsVisitCount.java"),
+        verifyWithInlineConfigParser(
+                getPath("InputAbstractJavadocNonTightHtmlTagsVisitCount.java"),
                 expected);
     }
 
     @Test
-    public void testVisitCountForCheckAcceptingJavadocWithNonTightHtml()
-            throws Exception {
-        final DefaultConfiguration checkConfig =
-                createModuleConfig(NonTightHtmlTagTolerantCheck.class);
-        checkConfig.addProperty("violateExecutionOnNonTightHtml", "true");
-        checkConfig.addProperty("reportVisitJavadocToken", "true");
+    public void testVisitCountForCheckAcceptingJavadocWithNonTightHtml() throws Exception {
         final String[] expected = {
```

Comment:
Why did we lose massive amount of violations?

Reply:
restored.
run into https://github.com/checkstyle/checkstyle/issues/13307, improved a bit.
Input was split by demand of our validation.